add content for INDEXMISSING and INDEXEMPTY (#412)
redis/docs: content/develop/interact/search-and-query/indexing/_index.md

[`FT.AGGREGATE`]({{< baseurl >}}/commands/ft.aggregate/) queries require `attribute` modifiers. Don't use JSONPath expressions in queries, except with the `LOAD` option, because the query parser doesn't fully support them.
{{% /alert %}}

> ## Index Missing or Empty Values

Reviewer: ```suggestion
## Index missing or empty values
```

> You can search for properties that do not exist in a given document (missing) with the INDEXMISSING option for FT.CREATE in conjunction with the ISMISSING function.  You can search for properities that exist but have no value (empty) by using INDEXEMPTY option with FT.CREATE.  Both query types require dialect 2.  Examples below:

Reviewer: ```suggestion
You can search for missing properties, that is, properties that do not exist in a given document, using the `INDEXMISSING` option to `FT.CREATE` in conjunction with the `ISMISSING` query function with `FT.SEARCH`. You can search for existing properties with no value (i.e., empty) using the `INDEXEMPTY` option with `FT.CREATE`. Both query types require DIALECT 2. Examples below:
```
